.github/workflows/ci.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - src
+  schedule:
```

Comment:
Do we want this? It might be confusing for the users if the manual includes features that aren't available in the stable release yet. I think we should either have nightly and stable versions, or just stable.

Reply:
Yes, once a week or manually triggered as part of a release seems preferable. I wonder if we can easily setup some form cross repo triggers? (that is once a stable releases CI run finished, trigger CI here?)

Reply:
Maybe let's add `workflow_dispatch` too, so we can trigger it manually.

Reply:
Neat, I didn't know you could do that. Added.